### PR TITLE
feat: checkout content adjustement (not full rework yet)

### PR DESCRIPTION
**Pull request by NeOMakinG**
## How does it look?
https://www.awesomescreenshot.com/video/8042555?key=6b104d500f9d58909ac56ecb1ba7edc9

### CODE REVIEW

File: templates/components/checkout-steps.tpl [hunk 1]
```diff
@@ -82,7 +82,7 @@
         </p>
       </div>
 
-      <div class="{$componentName}__step" data-step="2">
+      <div class="{$componentName}__step" data-step="checkout-addresses-step">
```
Comment (Hlavtox) [inline templates/components/checkout-steps.tpl:85]: @NeOMakinG Is this correct? The d-none thing on steps

Comment (NeOMakinG) [reply templates/components/checkout-steps.tpl:85]: The d-none? Steps are hidden if it's not the current and it's toggled using JS if you switch to another one

Comment (Hlavtox) [reply templates/components/checkout-steps.tpl:85]: Yes, but why is specifically the second one displayed by default?

Comment (NeOMakinG) [reply templates/components/checkout-steps.tpl]: Did hide every steps by default now

**Review by matthieu-rolland — COMMENTED** [review]
One question, lgtm otherwise 👍

**Comment by NeOMakinG** [comment]
It's now ready for review

I would like to pass some time with @Hlavtox on this in order to check that the markup is fine enough, pretty sure we can find some improvements on it!

**Comment by NeOMakinG** [comment]
As we're currently peer programming on it with Hlavtox, we're merging this, if you want to review it, do it, it will be taken under consideration inside another PR

**Pull request merged**